### PR TITLE
burger-items-memu-added

### PR DESCRIPTION
**Pull request by Farah7Tawfeek**
Burger Menu

### CODE REVIEW

File: assets/styles/style.css [hunk 1]
```diff
@@ -2,15 +2,25 @@
...
+    font-family: 'Fira Sans';
+    font-style: normal;
+    font-weight: 200;
+    font-display: swap;
+    src: url("../fonts/FiraSans-Medium.ttf") format('woff2');
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+
+html, 
 body{
```
Comment (AhmedBHameed) [inline assets/styles/style.css]:
```suggestion
body {
```

File: assets/styles/style.css [hunk 2]
```diff
@@ -24,5 +34,62 @@ body{
...
\ No newline at end of file
+    color:white;
+    display: flex;
+}
+
+.icon {
+    padding-right:4px;
+}
+
+.slide-menu{
```
Comment (AhmedBHameed) [inline assets/styles/style.css]:
```suggestion
.slide-menu {
```

File: index.html [hunk 3]
```diff
@@ -13,8 +14,24 @@
         </div>
             
         <div class="menu">
-            Menu
+            <div>
+            <i class="fas fa-bars icon"></i>
+            </div>
```
Comment (AhmedBHameed) [inline index.html]: Making every html tag in one indent can cause a lot of frustration in the future. For the moment it is a good practice to train your hand to write clean code without extensions.

```suggestion
            <div>
            	<i class="fas fa-bars icon"></i>
            </div>
```

File: index.html [hunk 4]
```diff
@@ -13,8 +14,24 @@
...
+        <div class="slide-menu active">
+            <div class="close-menu">
+                <i class="fas fa-times"></i>
+            </div>
+            <ul class="list">
+                <li class="item">Home</li>
+                <li class="item">Elements</li>
+                <li class="item">Generic</li>
+            </ul>
         </div>
```
Comment (AhmedBHameed) [inline index.html]: Always separate containers with new line. That will increase code readability.
```suggestion
        <div class="slide-menu active">
            <div class="close-menu">
                <i class="fas fa-times"></i>
            </div>
            
            <ul class="list">
                <li class="item">Home</li>
                <li class="item">Elements</li>
                <li class="item">Generic</li>
            </ul>
        </div>
```

**Review by AhmedBHameed — CHANGES_REQUESTED** [review]
Few Nit-picky.

**Please remove `Fira_Sans` folder.**

File: index.html (hunk 4 above)
Comment (Farah7Tawfeek) [reply index.html]: @AhmedBHameed I fixed the issues, shall I squash and merge now?

Comment (AhmedBHameed) [reply index.html]: @Farah7Tawfeek 

You still have fonts folder. After you remove it, you are ready to `squash and merge`.

**Review by AhmedBHameed — APPROVED** [review]
@Farah7Tawfeek 

You still have fonts folder. You can also review your code when you go to `Files changed`

Hey, after removing fonts, you are ready to `squash and merge`.

**Pull request merged**